src/pint/modelutils.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from pint.models.astrometry import AstrometryEquatorial, AstrometryEcliptic
+
+
+def convert_to_equatorial(model, force=False):
```

Comment:
I think a better name is "pulsar_ecl_to_equatorial" or "pulsar_ecliptic_to_equatorial" if this function only handles from pulsar ecl to equatorial. If David's galactic coordinate PR get merged, the user will get confused about this function.

Reply:
To be clear, I'm going with `model_x_to_y` (where x and y stand for equatorial and ecliptic, or vice versa).